axelrod/strategies/axelrod_second.py

```diff
@@ -199,3 +202,212 @@ def strategy(self, opponent: Player) -> Action:
+        When FD = 2:                                                                                                                                                                                                                                                    
+                                                                                                                                                                                                                                                                        
+            Set FD = 0                                                                                                                                                                                                                                                  
+            Set the variable 'NO':                                                                                                                                                                                                                                      
+            NO = ((NO * NK) + 3 - (3 * J) + (2 * JA) - (JA * J) / (NK + 1)                                                                                                                                                                                              
+            Where JA and J are the last moves of the strategy and the opponent (C=0, D=1)                                                                                                                                                                       
+            Increase the value of NK by 1                                                                                                                                                                                                                               
+            Cooperate                                                                                                                                                                                                                                                   
+                                                                                                                                                                                                                                                                        
+    Tranquilizer came in 27th place in Axelrod's second torunament. 
+   
+
+    Names:
+
+    - Tranquilizer: [Axelrod1980]_
+    """
+
+    name = 'Tranquilizer'
+    classifier = {
+        'memory_depth': float('inf'),
+        'stochastic': True,
+        'makes_use_of': {"game"},
+        'long_run_time': False,
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+
+    # Initialised atributes
+    def __init__(self):
+        super().__init__()
+        self.num_turns_after_good_defection = 0
```

Comment:
Could we add an inline comment here saying: `# this corresponds to the FD variable in the original fortran code`.